src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+    await db.schema
+        .createTable('member_user_public_key')
+        .addColumn('id', 'uuid', (col) => col.defaultTo(sql`v7uuid()`).primaryKey())
+        .addColumn('user_id', 'uuid', (col) => col.notNull())
+        .addColumn('value', 'text', (col) => col.notNull())
+        .addColumn('fingerprint', 'text', (col) => col.notNull())
```

Comment:
I believe the naming here is consistent with other tables where it's `{table noun} - {attribute}` like `study-run-status`. So here we have `member-user-public-key-value` and `member-user-public-key-fingerprint`. This is in contrast to, say, naming the `value` column `key` as that would be `member-user-public-key-key` which seems inconsistent to me :man_shrugging: .